### PR TITLE
Aml 3775 add error to mou

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

**Comment by Winsto** [comment]
Should this also be [compared to base other than master](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/compare/AML-3574-legal-agreement...AML-3775_AddErrorToMoU) ?

I have just had a look at that diff and generally looks ok, but seems as though a whole new view is added when the user story is only talking about adding an error display on no options chosen

**Comment by MichaelYoung1981** [comment]
> Should this also be [compared to base other than master](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/compare/AML-3574-legal-agreement...AML-3775_AddErrorToMoU) ?
> 
> I have just had a look at that diff and generally looks ok, but seems as though a whole new view is added when the user story is only talking about adding an error display on no options chosen

Sorry, meant to explain on the slack channel but forgot!  Slightly complicated, this is based on 3774 but also has changes from 3764 merged in!  Good luck getting git to work out that diff!

**Comment by Howesy77** [comment]
We have merge conflicts

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs [hunk 1]
```diff
@@ -132,13 +132,20 @@ public async Task<ActionResult> SignAgreement(GetEmployerAgreementRequest reques
...
-            if (choice != ReviewAgreementLater)
+            if (choice == ReviewAgreementLater)
             {
                 return RedirectToAction(ControllerConstants.IndexActionName, ControllerConstants.EmployerTeamControllerName);
             }
 
             var userInfo = OwinWrapper.GetClaimValue(ControllerConstants.UserRefClaimKeyName);
             var agreement = await _orchestrator.GetById(agreementId, hashedAccountId, userInfo);
+
+            if (choice == null)
```
Comment (Howesy77) [inline src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: Would it be better to have choice default to 0 so a const can be used to define the state?

Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: Isn’t the default value passed for a form field with no value null? I’m not sure zero would work and it’d be inconsistent with all other pages.

**Pull request merged**